graphqlapi/src/main/java/io/stargate/graphql/web/resources/GraphqlCache.java

```diff
@@ -241,6 +259,12 @@ public void onKeyspaceChanged(
       this.keyspace = keyspace;
     }
 
+    DmlGraphqlHolder(SchemaSource source, Keyspace keyspace, GraphQL graphQL) {
+      this.source = source;
+      this.graphqlFuture.complete(graphQL);
+      this.keyspace = keyspace;
```

Comment:
We don't need the keyspace here, it's only used by `init()`, which will never be called if this constructor was used. You can remove the constructor argument, that will also simplify the `get()` code.

You can just set the field to `null` for now. The class is getting a bit confusing though, I think I'll revisit it in a follow-up PR.

Reply:
done

Reply:
done